purchase_order_import/__manifest__.py

```diff
@@ -3,14 +3,20 @@
 
 {
     "name": "Purchase Order Import",
-    "version": "10.0.2.0.0",
+    "version": "14.0.1.0.0",
     "category": "Purchase Management",
     "license": "AGPL-3",
     "summary": "Update RFQ via the import of quotations from suppliers",
     "author": "Akretion,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/edi",
-    "depends": ["purchase", "base_business_document_import_stock", "onchange_helper"],
+    "depends": [
+        "base_business_document_import",
+        "onchange_helper",
+        "purchase",
```

Comment:
`"purchase"` is already in the dependency chain because of `"purchase_stock"`

Reply:
You're right! Fxied